<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Home.</title>

    <!-- Bootstrap Core CSS -->
    <link href="css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="css/style.css" rel="stylesheet">

    <!-- Custom Fonts -->
    <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
    <link href='https://fonts.googleapis.com/css?family=Kaushan+Script' rel='stylesheet' type='text/css'>
    <link href='https://fonts.googleapis.com/css?family=Droid+Serif:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
    <link href='https://fonts.googleapis.com/css?family=Roboto+Slab:400,100,300,700' rel='stylesheet' type='text/css'>

    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

    <link rel="apple-touch-icon" sizes="57x57" href="favicons/apple-touch-icon-57x57.png">
<link rel="apple-touch-icon" sizes="60x60" href="favicons/apple-touch-icon-60x60.png">
<link rel="apple-touch-icon" sizes="72x72" href="favicons/apple-touch-icon-72x72.png">
<link rel="apple-touch-icon" sizes="76x76" href="favicons/apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" sizes="114x114" href="favicons/apple-touch-icon-114x114.png">
<link rel="apple-touch-icon" sizes="120x120" href="favicons/apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" sizes="144x144" href="favicons/apple-touch-icon-144x144.png">
<link rel="apple-touch-icon" sizes="152x152" href="favicons/apple-touch-icon-152x152.png">
<link rel="apple-touch-icon" sizes="180x180" href="favicons/apple-touch-icon-180x180.png">
<link rel="icon" type="image/png" href="favicons/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="favicons/android-chrome-192x192.png" sizes="192x192">
<link rel="icon" type="image/png" href="favicons/favicon-96x96.png" sizes="96x96">
<link rel="icon" type="image/png" href="favicons/favicon-16x16.png" sizes="16x16">
<link rel="manifest" href="favicons/manifest.json">
<link rel="shortcut icon" href="favicons/favicon.ico">
<meta name="msapplication-TileColor" content="#333333">
<meta name="msapplication-TileImage" content="favicons/mstile-144x144.png">
<meta name="msapplication-config" content="favicons/browserconfig.xml">
<meta name="theme-color" content="#ffffff">

</head>

<body id="page-top" class="index">

    <!-- Navigation -->
    <nav class="navbar navbar-default navbar-fixed-top">
        <div class="container">
            <!-- Brand and toggle get grouped for better mobile display -->
            <div class="navbar-header page-scroll">
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                    <span class="sr-only">Toggle navigation</span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>
                <a class="navbar-brand page-scroll" href="#page-top">Home.</a>
            </div>

            <!-- Collect the nav links, forms, and other content for toggling -->
            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                <ul class="nav navbar-nav navbar-right">
                    <li class="hidden">
                        <a href="#page-top"></a>
                    </li>
                    <li>
                        <a class="page-scroll" href="#about">about.</a>
                    </li>
                    <li>
                        <a class="page-scroll" href="#pub">publications.</a>
                    </li>
                </ul>
            </div>
            <!-- /.navbar-collapse -->
        </div>
        <!-- /.container-fluid -->
    </nav>

    <!-- Header -->
    <header>
        <div class="container">
            <div class="intro-text">    
                <div class="intro-lead-in">Lydia Hopp</div>
                <div class="intro-heading">IZBI . Leipzig University . Härtelstr. 16 – 18 . D-04107 Leipzig . TEL: +49 341 97 16697 .<br><br><a href="mailto:hopp@izbi.uni-leipzig.de" >hopp@izbi.uni-leipzig.de</a> </div>
		
                <!--<a href="#about" class="page-scroll btn btn-xl">Zum Angebot</a>-->
		<img src="img/lydia.jpg" class="img-responsive img-circle img-centered" width="300" height="500" alt="">                                                                                            
			<div class="intro-heading"><br><br>GROUPS: <br><br><a href="http://www.izbi.uni-leipzig.de/science/research-groups/binder/">Omics bioinformatics for health and systems biology</a> <br><br> <a href="http://www.izbi.uni-leipzig.de/science/research-groups/loffler-wirth/">Algorithms for complex data in life sciences</a>
        </div>
    </header>

    <!-- Bilder Bereich -->
    <section id="about">
        <div class="container">
            <div class="row">
                <div class="col-lg-12 text-center">
                    <h2 class="section-heading">About.</h2>
					</div>  
                    <h3 class="section-subheading text-muted">Background.</h3>    
    <p>I studied biomathematics at the Ernst Moritz Arndt University in Greifswald including a semester abroad in Palmerston North, New Zealand. In November 2009 I started working at the Interdisciplinary Center for Bioinformatics (IZBI) where I wrote my diploma thesis entitled 'Gene expression in artificial genomes after perturbations: gene insertions and knock-outs'. After graduation in October 2010 I continued to work in the group of Hans Binder (IZBI). In October 2011 I joined the ESF II junior research group as a PhD student. Since 2017 I hold a doctorate in Informatics. </p>

		    <h3 class="section-subheading text-muted">Interest.</h3> 
<p>The main focus of my studies is understanding systems biology mainly in the context of diseases such as cancer. 
Novel high-throughput technologies are challenging bioinformaticians and data analysts nowadays, producing an incredible amount of highdimensional molecular data and demanding for powerful analysis tools.
Therefore we make use of maschine learning in terms of self organizing maps combined with methods of big data analysis (clustering, feature selection, reduction of data space, subtyping, enrichment methods).
Additionally, I developed a new integrative method for the combined analysis of several data entities. </p>
<p>One key area of my research is epigenetics and epigenetic regulation. An important aspect of epigenetic regulation is chromatin structure. 
It is determined by the activity of histone modifiers whose expression depend on environment, genetic factors as well as ageing. 
With the help of modeling and analysis of high dimensional data (gene expression, ChIP-Seq) associated epigenetic and genetic regulatory mechanisms can be studied.</p>


<img src="img/Graduation-icon.png" class="img-responsive img-circle" width="100" height="100" alt="">   
<a href="http://nbn-resolving.de/urn:nbn:de:bsz:15-qucosa2-163551">PhD thesis</a> </div>  

 </section>



    <!-- Über Mich Bereich -->
    <section id="pub" class="bg-light-gray">
        <div class="container">
            <div class="row">
                <div class="col-lg-12 text-center">
                    <h2 class="section-heading">Publications.</h2>
			  <div class="col-lg-12 text-left">
                  <!--  <h3 class="section-subheading text-muted">Lorem ipsum dolor sit amet consectetur.</h3> -->

 <h3>2019 (2)</h3>
  <ul>
    <li>
		H. Binder, E. Willscher, H. Loeffler-Wirth, L. Hopp, et al. <strong>DNA methylation, transcriptome and genetic copy number signatures of diffuse cerebral WHO grade II/III gliomas resolve cancer heterogeneity and development. </strong> <em> Acta Neuropathologica Communications. </em> 2019.</a> 
    </li><li>
        H. Loeffler-Wirth, M. Kreuz, L. Hopp, et al.<strong> A holistic view on the expression landscape of mature aggressive B-cell lymphoma. </strong> <em> Genome Medicine.</em> 2019. Accepted. </a>
    </li>
  </ul>
				  
  <h3>2018 (4)</h3>
  <ul>
    <li>
	L. Hopp, H. Loeffler-Wirth, L. Nersisyan, A. Arakelyan, H. Binder. <strong>Footprints of Sepsis Framed Within Community Acquired Pneumonia in the Blood Transcriptome. </strong> <em> Front. Immunol.</em> 2018. 9:1620, <a href="https://www.frontiersin.org/articles/10.3389/fimmu.2018.01620/full">doi: 10.3389/fimmu.2018.01620</a> 
    </li><li>
        M. Kunz,  H. Loeffler-Wirth, M. Dannemann, E. Willscher, G. Doose, J. Kelso, T. Kottek, B. Nickel, L. Hopp, J. Landsberg, S. Hoffmann, T. Tüting, P. Zigrino, C. Mauch, J. Utikal, M. Ziemer, H. J. Schulze, M. Hölzel, A. Roesch, S. Kneitz, S. Meierjohann, A. Bosserhoff, H. Binder, M. Schartl.<strong> RNA-seq analysis identifies different transcriptomic types and developmental trajectories of primary melanomas. </strong> <em>Oncogene.</em> 2018, <a href="https://www.nature.com/articles/s41388-018-0385-y">doi: 10.1038/s41388-018-0385-y </a>
    </li><li>
		T. Thalheim, L. Hopp, H. Binder, G. Aust, J. Galle. <strong>On the Cooperation between Epigenetics and Transcription Factor Networks in the Specification of Tissue Stem Cells.  </strong> <em> Epigenomes.</em> 2018. 2:20, <a href="https://www.mdpi.com/2075-4655/2/4/20/htm">doi: 10.3390/epigenomes2040020</a> 
	</li><li>
        L. Hopp, H. Loeffler-Wirth, J. Galle, H. Binder. <strong>Combined SOM-portrayal of gene expression and DNA mehtylation disentangles modes of epigenetic regulation in glioma. </strong> <em>Epigenomics.</em> 2018. 10:6, 745-764, <a href="https://www.futuremedicine.com/doi/10.2217/epi-2017-0140">doi: 10.2217/epi-2017-0140</a>  
   </li>
  </ul>

  <h3>2017 (6)</h3>
  <ul>
    <li>
      J. Przybilla, L. Hopp, M. Lubbert, M. Loeffler, J. Galle. <strong>Targeting DNA hypermethylation: Computational modeling of DNA demethylation treatment of acute myeloid leukemia.</strong> <em>Epigenetics.</em> 2017. 12: 886-896, <a href="https://www.tandfonline.com/doi/abs/10.1080/15592294.2017.1361090">doi: 10.1080/15592294.2017.1361090</a>  
   </li><li>
      K. Rohde, M. Klos, L. Hopp, X. Liu, M. Keller, M. Stumvoll, A. Dietrich, M. R. Schon, D. Gartner, T. Lohmann, M. Dressler, P. Kovacs, H. Binder, M. Bluher, Y. Bottcher. <strong>IRS1 DNA promoter methylation and expression in human adipose tissue are related to fat distribution and metabolic traits.</strong> <em>Sci. Rep.</em> 2017. 7: 12369, <a href="https://www.nature.com/articles/s41598-017-12393-5">doi: 10.1038/s41598-017-12393-5</a>  
   </li><li>
      M. Keller, L. Hopp, X. Liu, T. Wohland, K. Rohde, R. Cancello, M. Klös, K. Bacos, M. Kern, F. Eichelmann, A. Dietrich, M. R. Schön, D. Gärtner, T. Lohmann, M. Dreßler, M. Stumvoll, P. Kovacs, A. DiBlasio, C. Ling, H. Binder, M. Blüher, Y. Böttcher. <strong>Genome-wide DNA promoter methylation and transcriptome analysis in human adipose tissue unravels novel candidate genes for obesity.</strong> <em>Molecular Metabolism.</em> 2017. 6: 86-100., <a href="https://www.sciencedirect.com/science/article/pii/S2212877816302757">doi: 10.1016/j.molmet.2016.11.003</a>
   </li><li>
      Z. Hamidouche, K. Rother, J. Przybilla, A. Krinner, D. Clay, L. Hopp, C. Fabian, A. Stolzing, H. Binder, P. Charbord, J. Galle. <strong>Bistable Epigenetic States Explain Age-Dependent Decline in Mesenchymal Stem Cell Heterogeneity.</strong> <em>Stem Cells.</em> 2017. 35: 694-704, <a href="https://stemcellsjournals.onlinelibrary.wiley.com/doi/abs/10.1002/stem.2514">doi: 10.1002/stem.2514</a>  
   </li><li>
      T. Gerber, E. Willscher, H. Loeffler-Wirth, L. Hopp, D. Schadendorf, M. Schartl, U. Anderegg, G. Camp, B. Treutlein, H. Binder, M. Kunz. <strong>Mapping heterogeneity in patient-derived melanoma cultures by single-cell RNA-seq.</strong> <em>Oncotarget.</em> 2017. 8: 846-862, <a href="http://www.oncotarget.com/index.php?journal=oncotarget&page=article&op=view&path[]=13666&pubmed-linkout=1">doi: 10.18632/oncotarget.13666</a>  
   </li><li>
      H. Binder, L. Hopp, M. R. Schweiger, S. Hoffmann, F. Juhling, M. Kerick, B. Timmermann, S. Siebert, C. Grimm, L. Nersisyan, A. Arakelyan, M. Herberg, P. Buske, H. Loeffler-Wirth, M. Rosolowski, C. Engel, J. Przybilla, M. Peifer, N. Friedrichs, G. Moeslein, M. Odenthal, M. Hussong, S. Peters, S. Holzapfel, J. Nattermann, R. Hueneburg, W. Schmiegel, B. Royer-Pokora, S. Aretz, M. Kloth, M. Kloor, R. Buettner, J. Galle, M. Loeffler. <strong>Genomic and transcriptomic heterogeneity of colorectal tumours arising in Lynch syndrome.</strong> <em>J Pathol.</em> 2017. 243: 242-254, <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/path.4948">doi: 10.1002/path.4948</a>
   </li>
  </ul>

  <h3>2016 (1)</h3>
  <ul>
    <li>
      T. Gerber, E. Willscher, H. Loeffler-Wirth, L. Hopp, D. Schadendorf, M. Schartl, U. Anderegg, G. Camp, B. Treutlein, H. Binder, M. Kunz. <strong>Mapping heterogeneity in patient-derived melanoma cultures by single-cell RNA-seq.</strong> <em>Oncotarget.</em> 2016, <a href="http://www.oncotarget.com/index.php?journal=oncotarget&page=article&op=view&path[]=13666&path[]=43404">doi: 10.18632/oncotarget.13666</a>
   </li>
  </ul>

  <h3>2015 (4)</h3>
  <ul>
    <li>
      L. Hopp, E. Willscher, H. Löffler-Wirth, H. Binder. <strong>Function Shapes Content: DNA-Methylation Marker Genes and their Impact for Molecular Mechanisms of Glioma.</strong> <em>Journal of Cancer Research Updates.</em> 2015. 11/2015: 127-148, <a href="http://www.lifescienceglobal.com/pms/index.php/jcru/article/view/3420">doi: 10.6000/1929-2279.2015.04.04.1</a>
   </li><li>
      L. Hopp, L. Nersisyan, H. Löffler-Wirth, A. Arakelyan, H. Binder. <strong>Epigenetic Heterogeneity of B-Cell Lymphoma: Chromatin Modifiers.</strong> <em>Genes (Basel).</em> 2015. 6: 1076-112, <a href="http://www.mdpi.com/2073-4425/6/4/1076">doi: 10.3390/genes6041076</a>  
   </li><li>
      L. Hopp, H. Löffler-Wirth, H. Binder. <strong>Epigenetic Heterogeneity of B-Cell Lymphoma: DNA Methylation, Gene Expression and Chromatin States.</strong> <em>Genes (Basel).</em> 2015. 6: 812-40, <a href="http://www.mdpi.com/2073-4425/6/3/812/xml">doi: 10.3390/genes6030812</a>  
   </li><li>
      H. Binder, L. Hopp, K. Lembcke, H. Wirth. <strong>Personalized disease phenotypes from massive omics data.</strong>   In: <em>Big Data Analytics in Bioinformatics and Healthcare, IGI Global  </em>   2015.   359-378.  Ed: Wang Baoying and Li Ruowang and Perrizo William, <a href="https://www.igi-global.com/chapter/personalized-disease-phenotypes-from-massive-omics-data/121465">doi: 10.4018/978-1-4666-6611-5.ch015</a> 
   </li>
  </ul>

  <h3>2014 (2)</h3>
  <ul>
   <li>
      H. Wirth, M.V. Çakir, L. Hopp, H. Binder. <strong>Analysis of MicroRNA Expression Using Machine Learning.</strong>   In: <em>miRNomics: MicroRNA Biology and Computational Analysis, Humana Press  </em>   2014. 1107: 257-278.  Ed: Malik Yousef and Jens Allmer, <a href="">doi: </a> 
   </li><li>
      Mehmet Volkan Çakir, Henry Wirth, Lydia Hopp, Hans Binder. <strong>MicroRNA Expression Landscapes in Stem Cells, Tissues, and Cancer.</strong> In: <em>miRNomics: MicroRNA Biology and Computational Analysis, Humana Press  </em> 2014. 1107: 279-302.  Ed: Malik Yousef and Jens Allmer, <a href="https://link.springer.com/protocol/10.1007/978-1-62703-748-8_17">doi: 10.1007/978-1-62703-748-8_17</a>
   </li>
  </ul>

  <h3>2013 (2)</h3>
  <ul>
    <li>
      L. Hopp, H. Wirth, M. Fasold, H. Binder. <strong>Portraying the expression landscapes of cancer subtypes: a glioblastoma multiforme and prostate cancer case study.</strong> <em>Systems Biomedicine.</em> 2013 , <a href="https://www.tandfonline.com/doi/abs/10.4161/sysb.25897">doi: 10.4161/sysb.25897</a>
   </li><li>
      L. Hopp, K. Lembcke, H. Binder, H. Wirth. <strong>Portraying the Expression Landscapes of B-Cell Lymphoma-Intuitive Detection of Outlier Samples and of Molecular Subtypes.</strong> <em>Biology.</em> 2013. 2: 1411-1437, <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4009791/">doi: 10.3390/biology2041411</a>  
   </li>
  </ul>

  <h3>2012 (1)</h3>
  <ul>
    <li>
      L. Steiner, L. Hopp, H. Wirth, J. Galle, H. Binder, S. J. Prohaska, T. Rohlf. <strong>A global genome segmentation method for exploration of epigenetic patterns.</strong> <em>PLoS One.</em> 2012. 7: e46811, <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0046811">doi: 10.1371/journal.pone.0046811</a>  
   </li>
  </ul>

  <h3>2011 (2)</h3>
  <ul>
    <li>
      H. Binder, M. Fasold, L. Hopp, V. Cakir, M. von Bergen, H. Wirth. <strong>Genomic and molecular Phenotypic Portraits - Exploring The "Omes" With Individual Resolution.</strong>   In: <em>6th International Symposium on Health Informatics and Bioinformatics (HIBIT) </em>   2011.  115-122.  Ed: IEEE Explore.
   </li><li>
      H. Binder, M. Fasold, L. Hopp, M.V. Cakir, L. G. Armstrong Nikkhah, A. Stolzing. <strong>Portraying high-dimensional -omics data with individual resolution.</strong>   In: <em>CAMDA </em>   2011.  
   </li>
  </ul>

    <style>
   
      #section-left {
        width: 35%;
        float: left;
        height: 100%;
        background-color: #669999;
        color: #fff;
        text-align: center;
      }
      #section-left .section {
        margin-top: 0%;
        padding: 1em;
      }
      #section-left .section .logo {
        height: 150px;
        width: 150px;
        border-radius: 50%;
        margin: 0 auto;
       /*! background: url('http://jaipurstudents.com/broker/wp-content/uploads/2015/10/dummy-logo-300x140.png');  */
      }
      .intro hr {
        width: 10em;
        display: block;
        height: 2px;
        border: 0px;
        border-top: 2px solid;
      }
      .dob {
        /*text-decoration: overline underline; */
	border-top: 1px solid #fff;
	padding-top: 3px;
	border-bottom: 1px solid #fff;
	padding-bottom: 3px;
      }
      .intro .content {
        margin-bottom: 20px;
      }
      #contact {
        margin-top: 40px;
      }
      h1 {
	font-family: Montserrat,"Helvetica Neue",Helvetica,Arial,sans-serif;
        text-transform: uppercase;
      }
      #section-right {
        width: 65%;
        float: right;
        /*height: 100%;*/
      }
      #section-right .section {
        margin: 0 0 30px 0;
      }
      .wrapper {
        padding: 2em;
      }
      .fa {
        margin-right: 15px;
      }
      .wrapper .title {
        color: #669999;
        font-size: 1.3em;
        font-variant: small-caps;
        letter-spacing: 0.1em;
        font-weight: bold;
        border-bottom: 2px solid #f2f2f2;
      }
      b:link {
        color: #fff;
        text-decoration: none;
      }
      b:visited {
        color: #fff;
        text-decoration: none;
      }     
      b:hover {
        color: #fff;
        text-decoration: none;
      }
      b:active {
        color: #fff;
        text-decoration: none;
      }      


      h4 {
        text-transform: uppercase;
        font-size: 1em;
      }
      h5 {
        font-size: 0.8em;
        text-transform: uppercase;
        font-weight: normal;
      }
    </style>
              
    </section>
   
 
    
    

    <!-- jQuery -->
    <script src="js/jquery.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="js/bootstrap.min.js"></script>

    <!-- Plugin JavaScript -->
    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.3/jquery.easing.min.js"></script>
    <script src="js/classie.js"></script>
    <script src="js/cbpAnimatedHeader.js"></script>

 
    <!-- Custom Theme JavaScript -->
    <script src="js/claudio.js"></script>

</body>

</html>
